docs/2.0/manual.md

```diff
@@ -437,22 +440,26 @@ Pheniqs constructs new segments for the output read and must generate correspond
+
+**I think this would be clearer if the key was "include only pf reads" or something like that. When I talk about filtered reads I generally refer to those that passed a particular filter, not those that did not pass.**
 
 # Configuration validation
-The `-V/--validate` command line flag makes Pheniqs evaluate the supplied instruction and emit a human readable description of the instruction without actually executing it. It is sometimes useful to inspect this description before executing to make sure all implicit parameters are allocated the desired values. To also print out the barcode distance metric for each closed class decoder you may additionally set the `-D/--distance` command line flag. The top half of the matrix, above the diagonal, is the pairwise Hamming distance, while the bottom half is the maximum number of correctable errors the pair can tolerate, known as the Shannon bound.
+The `-V/--validate` command line flag makes Pheniqs evaluate the supplied instruction and emit a human readable description of the instruction without actually executing it. It is sometimes useful to inspect this description before executing to make sure all implicit parameters are allocated the desired values. To also print out the barcode distance metric for each closed class decoder you may additionally set the `-D/--distance` command line flag. The top half of the matrix, above the diagonal, is the pairwise Hamming distance, while the bottom half is the maximum number of correctable errors the pair can tolerate, known as the [Shannon bound](https://en.wikipedia.org/wiki/Shannon%E2%80%93Hartley_theorem).
 
 # Demultiplexing statistics
 Pheniqs emits a comprehensive demultiplexing report with statistics about both inputs and outputs.
 
+**Will this be where we put Mohammed's MultiQC Report?**
+
 ## Input
 A quality statistics report for every segment in the input is provided in the `demultiplex input report` element.
 
 | JSON field                                      | Description
 | : --------------------------------------------- | :---------------------------------------------------------------------
 | **count**                                       | all input reads
 | **pf count**                                    | input reads that *passed vendor quality control*
-| **pf fraction**                                 | **pf count** / **count**
+| **pf fraction**                                 | **pf count** / ** (pf count + non-pf count)**
```

Comment:
items in bold are references to other variables in the table. since "non-pf count" is not a variable this is technically incorrect. "count" is simply all reads processed by the pipeline. Please see the revised table...